voluncheer/profiles/test_models.py

```diff
@@ -80,4 +84,43 @@ def test_organization_details(self):
     def test_volunteer_details(self):
         """Tests basic volunteer details."""
         self.assertEqual(self.luke.name, "Luke Skywalker")
+        self.assertEqual(self.luke.date_of_birth, "1955-09-25")
         self.assertEqual(self.vader.name, "Darth Vader")
+        self.assertEqual(self.vader.date_of_birth, "1931-01-17")
+
+    def test_admin_details(self):
+        """Tests admin account has proper attributes."""
+        self.assertTrue(self.admin.is_staff)
+        self.assertTrue(self.admin.is_superuser)
+        self.assertEqual(self.admin.type, UserType.ADMIN)
+
+    def test_email_is_required(self):
+        """Tests creating a user without an email raises desired ValueError"""
+        with self.assertRaisesMessage(
+            ValueError, "creating a user requires an email field"
+        ):
```

Comment:
The actual error message here is not particularly important, I'd prefer just asserting the correct error type is raised. Specifying the actual content is only really useful when it is dynamic. When the content is static the test serves as a change detector which is an anti-pattern.

Reply:
Thank you for pointing out, changed to assertRaises().